src/main/java/net/mcreator/ui/modgui/ArmorGUI.java

```diff
@@ -898,6 +921,10 @@ private void updateArmorTexturePreview() {
 		armor.bodySpecialInfo = StringUtils.splitCommaSeparatedStringListWithEscapes(bodySpecialInfo.getText());
 		armor.leggingsSpecialInfo = StringUtils.splitCommaSeparatedStringListWithEscapes(leggingsSpecialInfo.getText());
 		armor.bootsSpecialInfo = StringUtils.splitCommaSeparatedStringListWithEscapes(bootsSpecialInfo.getText());
+		armor.helmetImmuneToFire = helmetImmuneToFire.isSelected();
```

Comment:
You could disable these fields for 1.15.2 generator using field exclusions in yaml

Reply:
Done